cmf-cli/Commands/new/IoTCommand.cs

```diff
@@ -187,7 +188,7 @@ public void ExecuteV10(IDirectoryInfo workingDir, string version, string htmlPac
 
             iotCustomPackage.RelatedPackages = new()
             {
-                new RelatedPackage() { Path = fileSystem.DirectoryInfo.New(this.fileSystem.Path.GetRelativePath(iotCustomPackageWorkDir.FullName, fileSystem.DirectoryInfo.New(htmlPackageLocation).FullName)), PreBuild = true, PrePack = true }
+                new RelatedPackage() { Path = Path.Join(fileSystem.Path.GetRelativePath(iotCustomPackageWorkDir.FullName, fileSystem.DirectoryInfo.New(htmlPackageLocation).FullName)), PreBuild = true, PrePack = true }
```

Comment:
should use IO.Abstractions

Reply:
also, are these equivalent?

Reply:
maybe not, but I found a simple way to get this